app/assets/stylesheets/application.css.scss

```diff
@@ -119,3 +121,7 @@ a:hover {
 #footer {
     background:#000;
 }
+
+#username-link a, a:link, a:visited {
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line